inst/COPYRIGHTS

```diff
@@ -0,0 +1,26 @@
+Copyright for most of the included code is held by the Delphi group.
```

Comment:
> held by the Delphi group

Maybe?

Reply:
Wondering about the format for `inst/COPYRIGHT`.

Looks like there is no single standard format. We could do a free-form version, but it will be unwieldy and unreadable if we include lots of sources and licenses.
Some formats from other R packages that seem particularly appealing when listing multiple sources are:
- [Block for each library](https://github.com/zhengxwen/gdsfmt/blob/d82c4f0/inst/COPYRIGHTS) that includes the library's license and/or notes on usage
- [Debian package standard for copyright files](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0)
    - [Simple example](https://github.com/kaskr/adcomp/blob/master/TMB/inst/COPYRIGHTS)
    - [More complex example](https://github.com/trevorld/r-optparse/blob/f7d6363/inst/COPYRIGHTS)

Reply:
Since we only are including one external copyright/license, not going to worry about the format for now.

I'm going to go with option a) and not say anything about the majority of the code. Will open an issue for figuring out the copyright on our end.

Reply:
https://github.com/cmu-delphi/epiprocess/issues/348
https://github.com/cmu-delphi/epiprocess/issues/349